2024-06-24 19:44:26 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x115ccdd30>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:44:26 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x115ccdee0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:44:26 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x115dd30d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:44:26 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x115dd3700>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:44:26 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x115dd38b0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:44:26 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x115dd3a60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:47:47 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x116558fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:47:47 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x116587220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:47:47 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1165873d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:47:47 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x116587a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:47:47 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x116587bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:47:47 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x116587d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:58:17 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117edbfd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:58:17 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12090a220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:58:17 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12090a3d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:58:17 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12090aa00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:58:17 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12090abb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 19:58:17 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12090ad60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:00:46 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f46040>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:00:46 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f46250>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:00:46 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f46400>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:00:46 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f46a30>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:00:46 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f46be0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:00:46 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f46d90>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:04:53 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1189ecfd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:04:53 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x118a18220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:04:53 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x118a183d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:04:53 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x118a18a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:04:53 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x118a18bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:04:53 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x118a18d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:07:49 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1197bb070>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:07:49 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1197bb280>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:07:49 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1197bb430>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:07:49 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1197bba60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:07:49 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1197bbc10>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:07:49 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1197bbdc0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:10:50 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120c96df0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:10:50 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120cc8220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:10:50 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120cc83d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:10:50 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120cc8a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:10:50 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120cc8bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:10:50 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120cc8d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:11:50 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120297fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:11:50 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1202c5220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:11:50 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1202c53d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:11:50 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1202c5a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:11:50 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1202c5bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:11:50 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1202c5d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:16:46 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1155ca070>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:16:46 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1155ca280>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:16:46 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1155ca430>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:16:46 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1155caa60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:16:46 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1155cac10>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:16:46 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1155cadc0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:26:32 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x119edd040>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:26:32 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x119edd220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:26:32 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x119edd3d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:26:32 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x119edda00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:26:32 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x119eddbb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:26:32 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x119eddd60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:29:48 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x119ec1040>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:29:48 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x119ec1250>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:29:48 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x119ec1400>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:29:48 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x119ec1a30>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:29:48 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x119ec1be0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-24 20:29:48 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x119ec1d90>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 12:47:16 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120297fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 12:47:16 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1202c6220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 12:47:16 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1202c63d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 12:47:16 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1202c6a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 12:47:16 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1202c6bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 12:47:16 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1202c6d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 13:10:36 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107e7ffd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 13:10:36 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107eac220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 13:10:36 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107eac3d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 13:10:36 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107eaca00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 13:10:36 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107eacbb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 13:10:36 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107eacd60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 13:57:27 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11009bfd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 13:57:27 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1100ca220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 13:57:27 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1100ca3d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 13:57:27 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1100caa00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 13:57:27 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1100cabb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 13:57:27 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1100cad60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 14:00:56 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bf09070>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 14:00:56 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bf09280>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 14:00:56 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bf09430>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 14:00:56 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bf09a60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 14:00:56 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bf09c10>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 14:00:56 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bf09dc0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:29:01 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117683130>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:29:01 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117683340>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:29:01 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1176834f0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:29:01 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117683b20>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:29:01 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117683cd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:29:01 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117683e80>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:35:41 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x10fde8fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:35:41 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x10fe16220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:35:41 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x10fe163d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:35:41 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x10fe16a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:35:41 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x10fe16bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:35:41 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x10fe16d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:38:48 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117b2b070>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:38:48 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117b2b280>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:38:48 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117b2b430>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:38:48 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117b2ba60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:38:48 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117b2bc10>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:38:48 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117b2bdc0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:44:12 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117d92fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:44:12 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117dc1220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:44:12 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117dc13d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:44:12 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117dc1a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:44:12 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117dc1bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:44:12 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117dc1d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:51:49 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c2abfa0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:51:49 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c7091f0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:51:49 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c7093a0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:51:49 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c7099d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:51:49 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c709b80>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 15:51:49 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c709d30>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:24:26 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117517fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:24:26 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117544220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:24:26 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1175443d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:24:26 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117544a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:24:26 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117544bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:24:26 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117544d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:31:27 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12436bfd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:31:27 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12439b220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:31:27 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12439b3d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:31:27 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12439ba00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:31:27 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12439bbb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:31:27 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12439bd60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:33:02 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11d509070>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:33:02 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11d509280>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:33:02 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11d509430>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:33:02 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11d509a60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:33:02 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11d509c10>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:33:02 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11d509dc0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:34:08 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x122a990a0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:34:08 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x122a99220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:34:08 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x122a993d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:34:08 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x122a99a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:34:08 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x122a99bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:34:08 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x122a99d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:40:19 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bf6ff10>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:40:19 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bfa1220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:40:19 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bfa13d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:40:19 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bfa1a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:40:19 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bfa1bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:40:19 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bfa1d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:46:23 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x118345070>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:46:23 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1183451f0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:46:23 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1183453a0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:46:23 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1183459d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:46:23 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x118345b80>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:46:23 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x118345d30>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:49:00 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11b747fa0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:49:00 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11b7761f0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:49:00 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11b7763a0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:49:00 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11b7769d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:49:00 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11b776b80>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:49:00 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11b776d30>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:55:41 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f0a0a0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:55:41 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f0a220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:55:41 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f0a3d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:55:41 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f0aa00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:55:41 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f0abb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 16:55:41 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f0ad60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:14:06 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117b8bfd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:14:06 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117bb8220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:14:06 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117bb83d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:14:06 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117bb8a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:14:06 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117bb8bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:14:06 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117bb8d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:17:03 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x127684070>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:17:03 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x127684280>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:17:03 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x127684430>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:17:03 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x127684a60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:17:03 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x127684c10>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:17:03 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x127684dc0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:27:54 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x124f57fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:27:54 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x124f84220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:27:54 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x124f843d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:27:54 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x124f84a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:27:54 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x124f84bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:27:54 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x124f84d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:35:23 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11638ffd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:35:23 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1163bd220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:35:23 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1163bd3d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:35:23 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1163bda00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:35:23 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1163bdbb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:35:23 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1163bdd60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:37:42 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x122ec7070>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:37:42 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x122ec7280>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:37:42 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x122ec7430>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:37:42 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x122ec7a60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:37:42 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x122ec7c10>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:37:42 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x122ec7dc0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:40:14 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1193bbfd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:40:14 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1193e9220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:40:14 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1193e93d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:40:14 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1193e9a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:40:14 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1193e9bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:40:14 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1193e9d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:42:40 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12271e0a0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:42:40 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12271e220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:42:40 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12271e3d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:42:40 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12271ea00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:42:40 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12271ebb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:42:40 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x12271ed60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:45:06 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11a1d7fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:45:06 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c21d220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:45:06 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c21d3d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:45:06 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c21da00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:45:06 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c21dbb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:45:06 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c21dd60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:46:26 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f1a040>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:46:26 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f1a250>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:46:26 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f1a400>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:46:26 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f1aa30>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:46:26 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f1abe0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:46:26 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117f1ad90>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:47:11 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1156b3fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:47:11 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x116309220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:47:11 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1163093d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:47:11 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x116309a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:47:11 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x116309bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:47:11 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x116309d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:49:55 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x128009070>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:49:55 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x128009280>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:49:55 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x128009430>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:49:55 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x128009a60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:49:55 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x128009c10>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:49:55 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x128009dc0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:52:16 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x124d47fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:52:16 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x124d76220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:52:16 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x124d763d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:52:16 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x124d76a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:52:16 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x124d76bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:52:16 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x124d76d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:53:13 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x114848040>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:53:13 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x114848250>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:53:13 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x114848400>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:53:13 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x114848a30>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:53:13 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x114848be0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:53:13 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x114848d90>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:57:56 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c213fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:57:56 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c241220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:57:56 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c2413d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:57:56 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c241a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:57:56 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c241bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:57:56 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11c241d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:59:56 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x111ec8070>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:59:56 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x111ec8280>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:59:56 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x111ec8430>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:59:56 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x111ec8a60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:59:56 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x111ec8c10>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-25 17:59:56 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x111ec8dc0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:02:17 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120896fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:02:17 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1208c8220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:02:17 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1208c83d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:02:17 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1208c8a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:02:17 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1208c8bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:02:17 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1208c8d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:07:35 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11baa3fd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:07:35 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bad2220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:07:35 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bad23d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:07:35 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bad2a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:07:35 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bad2bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:07:35 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11bad2d60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:09:09 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120ac4070>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:09:09 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120ac4280>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:09:09 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120ac4430>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:09:09 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120ac4a60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:09:09 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120ac4c10>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:09:09 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x120ac4dc0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:15:25 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11a15bfd0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:15:25 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11a18f220>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:15:25 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11a18f3d0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:15:25 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11a18fa00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:15:25 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11a18fbb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-06-26 00:15:25 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x11a18fd60>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
